syncd/Syncd.cpp

```diff
@@ -4309,6 +4309,8 @@ void Syncd::onSwitchCreateInInitViewMode(
                 sai_serialize_object_id(switchVid).c_str(),
                 newHw.c_str());
 
+        currentHw = currentHw == "none"? "":currentHw;
```

Comment:
Please add a comment here explaining why this is needed, and that the behavior change is from SAI call.
Also a TODO to remove this change in the future if SAI vendor agrees to fix this at their end.

Reply:
Done.